lib/config/executor/network_interface.go

```diff
@@ -74,6 +75,9 @@ type ContainerNetwork struct {
 	// set of network wide links and aliases for this container on this network
 	Aliases []string `vic:"0.1" scope:"hidden" key:"aliases"`
 
+	// Level of trust configured for this network
+	TrustLevel common.TrustLevel
```

Comment:
Please add the base vic version annotation - you shouldn't need any of the others:
`vic:"0.1"`

This will inherit the scope defined for the ContainerNetwork struct which I presume is "read-only" - this value _should_ be read only.